README.md

```diff
@@ -69,7 +70,7 @@ A Unix timestamp (integer seconds since Jan 1, 1970 UTC). Required for all reque
 
 #### X-Authorization-Content-SHA256 Header
 
-The base64 encoded SHA-256 hash value used to generate the signature base string. This is analogous to the standard Content-MD5 header. Required for any request except GET or HEAD.
+The base64 encoded SHA-256 hash value used to generate the signature base string. This is analogous to the standard Content-MD5 header. Required for any request where Content-Length > 0.
```

Comment:
for prose, maybe better to say something like "where Content-Length is not 0 (for example a POST request with a body)".